### DISCUSSION TITLE
TuringCourses/short_term/data_visualization/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#198] [General]
# TuringCourses/short_term/data_visualization/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/short_term/data_visualization/

**Comment by CrazySpottedDove** [comment]
这里要注意，课程要求开发的可视化分析系统并不是一个泛用性的系统，应当基于选择的数据集定制特定功能，我们不慎做了个泛用性系统，被老师提问“你觉得你们做的项目和tableau有什么区别”薄纱了QAQ
